src/app/[...404]/page.tsx

```diff
@@ -45,10 +45,10 @@ const Custom404Page = () => {
           className="bg-blue-600 hover:bg-blue-700 text-white font-semibold py-2 px-6 rounded-full transition-colors duration-300"
         >
           <Link href="/">
-            <a className="flex items-center">
```

Comment:
You should never use `<a>` inside a `<Link>` in NextJs

Reply:
Thanks for flagging, i missed it while reviewing